### PR TITLE
Add/unraid net

### PR DESCRIPTION
**Pull request by zackspear**
Base copy additions for Unraid.net additions. Added to translations.txt since our additions appear on every page.

### CODE REVIEW

File: translations.txt [hunk 1]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Server Offline=
+Remote access=
+Access unavailable=
+Opens new HTTPS window to %s=
+Sign In to resolve=
+Sign In to Purchase Key=
+Purchase Key or Extend Key=
+No Keyfile=
+Thanks for supporting Unraid=Thanks for supporting Unraid!
+Lets unleash your hardware=Let's unleash your hardware!
```
Comment (Squidly271) [inline translations.txt:450]: Per design guide, ! is not allowed on the left

File: translations.txt [hunk 2]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Local access=
+Server Offline=
+Remote access=
+Access unavailable=
+Opens new HTTPS window to %s=
+Sign In to resolve=
+Sign In to Purchase Key=
+Purchase Key or Extend Key=
+No Keyfile=
+Thanks for supporting Unraid=Thanks for supporting Unraid!
```
Comment (Squidly271) [inline translations.txt:449]: Per design guide, ! is not allowed on the left

File: translations.txt [hunk 3]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Trial Expired=
+Your Trial has expired=
+To continue using Unraid OS you may purchase a license key=
+You have used all your Trial extensions=You have used all your Trial extensions.
+Alternately, you may request a Trial extension=
+Basic=
+Plus=
+Pro=
+GUID Error=
+Registration key  GUID mismatch=Registration key / GUID mismatch
```
Comment (Squidly271) [inline translations.txt:468]: Double spaces don't work.  Need to be compressed to one on the left.  Right side can have doubles

File: translations.txt [hunk 4]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Stale Server=
+Please refresh the page to ensure you load your latest configuration=
+Registration Wizard=
+To Registration Wizard Home=
+Continue Trial=
+Continue to Unraid=
+Toggle server info visibility=
+You can safely close this window=
+Automatically closing in=
+bye, bye 👋=
```
Comment (Squidly271) [inline translations.txt:502]: Untested with extended characters

File: translations.txt [hunk 5]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Remote access=
+Access unavailable=
+Opens new HTTPS window to %s=
+Sign In to resolve=
+Sign In to Purchase Key=
+Purchase Key or Extend Key=
+No Keyfile=
+Thanks for supporting Unraid=Thanks for supporting Unraid!
+Lets unleash your hardware=Let's unleash your hardware!
+You are all set 👍=
```
Comment (Squidly271) [inline translations.txt:451]: Untested with extended characters

File: translations.txt [hunk 6]
```diff
@@ -390,3 +390,232 @@ This will reboot the system=
...
+Basic=
+Plus=
+Pro=
+GUID Error=
+Registration key  GUID mismatch=Registration key / GUID mismatch
+The license key file does not correspond to the USB Flash boot device Please copy the correct key file to the /boot/config directory on your USB Flash boot device=The license key file does not correspond to the USB Flash boot device. Please copy the correct key file to the **/boot/config** directory on your USB Flash boot device
+or click Replace Key to transfer your registration to this new flash device=
+Missing key file=
+It appears that your license key file is corrupted or missing The key file should be located in the /boot/config directory on your USB Flash boot device If you do not have a backup copy of your license key file you may attempt to recover your key If this was a Trial installation, you may purchase a license key=It appears that your license key file is corrupted or missing. The key file should be located in the **/boot/config** directory on your USB Flash boot device. If you do not have a backup copy of your license key file you may attempt to recover your key. If this was a Trial installation, you may purchase a license key.
```
Comment (Squidly271) [inline translations.txt:472]: slashes on left

File: translations.txt (hunk 3 above)
Comment (zackspear) [reply translations.txt:468]: Tom mentioned that only **helptext.txt** is included by default for English. So I took that as needing to include the default text for my translations inside our PLG like so: `_('Registration key / GUID mismatch')`

Will the translation filtering automatically remove the space and the slash? or just the slash as I've already assumed?

Meaning can I get away with _just_ removing the extra space in this key?

File: translations.txt (hunk 4 above)
Comment (zackspear) [reply translations.txt:502]: It seems to work fine when I upload ZIPs with translations for German, French, & Spanish. But I will re-confirm.

File: translations.txt (hunk 6 above)
Comment (zackspear) [reply translations.txt:472]: Ahh thanks. Had just focused on the asterisks and glazed over the slashes

File: translations.txt (hunk 3 above)
Comment (Squidly271) [reply translations.txt:468]: I would think you need to remove both.   While I never tested it with space slash space the spec does say no double spaces.   I guess the order in which the parser does things dictates this.

TBH when I went through this chore in the plugins I tested each and every line with a sample translation.

File: translations.txt (hunk 4 above)
Comment (Squidly271) [reply translations.txt:502]: A ton easier IMO to simply add the appropriate line to say nl_NL with a sample translation and the delete the applicable .dot file.

Comment (zackspear) [reply translations.txt:502]: Confirming that it works with an emoji in the key.

Screen Shot 2020-12-08 at 09 08 08

**Pull request merged**